gensrc/script/doris_builtins_functions.py

```diff
@@ -114,12 +114,30 @@
     [['array'], 'ARRAY', ['ARRAY', '...'], '', '', '', '', ''],
     [['array'], 'ARRAY', ['MAP', '...'], '', '', '', '', ''],
     [['array'], 'ARRAY', ['STRUCT', '...'], '', '', '', '', ''],
-    [['%element_extract%'], 'VARCHAR', ['ARRAY', 'INT'],  '', '', '', '', ''],
-    [['%element_extract%'], 'VARCHAR', ['ARRAY', 'VARCHAR'],  '', '', '', '', ''],
     [['%element_extract%'], 'VARCHAR', ['MAP', 'VARCHAR'],  '', '', '', '', ''],
     [['%element_extract%'], 'VARCHAR', ['MAP', 'INT'],  '', '', '', '', ''],
     [['%element_extract%'], 'VARCHAR', ['STRUCT', 'INT'],  '', '', '', '', ''],
     [['%element_extract%'], 'VARCHAR', ['STRUCT', 'VARCHAR'],  '', '', '', '', ''],
+
+    [['element_at', '%element_extract%'], 'TINYINT', ['ARRAY_TINYINT', 'INT'],
```

Comment:
why there are 2 `%`?

Reply:
And do we support `array<largeint>`?

Reply:
%element_extract% maybe used to avoid user call directly